### PR TITLE
docs: update documentation

### PR DESCRIPTION
**Pull request by wenyanta**
Added Alder Lake and Raptor Lake to supported platforms. Updated kernel support. Updated ACRN reference links to v3.2.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -54,8 +54,10 @@ ACRN is supported on the following Intel platforms:
 * Whiskey Lake
 * Tiger Lake
 * Elkhart Lake
```
Comment (saininav) [inline docs/getting-started.md:56]: Just keep, tigerlake, adl and rpl only

File: docs/getting-started.md [hunk 2]
```diff
@@ -300,8 +302,10 @@ Supported Boards:
 - nuc11tnbi5
 - cfl-k700-i7
 - whl-ipc-i5
```
Comment (saininav) [inline docs/getting-started.md:304]: Update here too

File: docs/getting-started.md [hunk 3]
```diff
@@ -66,7 +68,7 @@ Make sure your build host meets the following requirements:
 - 50 Gbytes of free disk space
 - Runs a supported Linux distribution
 
-Currently, the Yocto Project is supported on a number of Linux Distributions. This guide covers only for Ubuntu 18.04 (LTS). To know more about the complete list of    supported Linux distributions, please visit [Supported Linux Distributions](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#detailed-supported-distros)
+Currently, the Yocto Project is supported on a number of Linux Distributions. This guide covers only for Ubuntu 22.04 (LTS). To know more about the complete list of    supported Linux distributions, please visit [Supported Linux Distributions](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#detailed-supported-distros)
 - Git 1.8.3.1 or greater
```
Comment (saininav) [inline docs/getting-started.md:72]: Update git, tar, gcc and others version based on latest requirement
https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution

File: docs/getting-started.md [hunk 4]
```diff
@@ -66,7 +68,7 @@ Make sure your build host meets the following requirements:
 - 50 Gbytes of free disk space
```
Comment (saininav) [inline docs/getting-started.md:68]: Update free space disk as per
https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution

File: docs/getting-started.md (hunk 2 above)
Comment (wenyanta) [reply docs/getting-started.md:304]: should I add nuc11tnbv7 after it is verified? this is Jing Hui's nuc11.

Comment (saininav) [reply docs/getting-started.md]: No

**Pull request merged**